reports/helmcharts/save-to-cosmos.py

```diff
@@ -22,7 +24,7 @@ def remove_documents(container):
         current_time = get_formatted_datetime()
         print(f"Removing all document added before {current_time}")
         for item in container.query_items(
-                query='SELECT * FROM c',
+                query = f"SELECT * FROM c WHERE c.environment = '{environment}'",
```

Comment:
May need to get job running first in PTL so that this line works: https://github.com/hmcts/cnp-flux-config/blob/master/apps/monitoring/version-reporter/helmcharts/helm-charts.yaml#L38

at the moment in cosmos this env key will not match at all